mlir/utils/jenkins/Jenkinsfile.miopen-test

```diff
@@ -106,15 +103,21 @@ mlir/utils/jenkins/miopen-tests/**\
                         }
                     }
                     stage("Test MIOpen config") {
+                        environment {
+                            // Make libMIOpen.so become accessible to the test driver
```

Comment:
I think you can drop 'become'.

Reply:
If I drop the verb then this won't be a complete sentence. At this case I see be or become as interchangeable so will keep the original comments.

Reply:
Sorry I just see this is imperative, will do.